website/views.py

```diff
@@ -339,12 +339,18 @@ def resolve_guid(guid, suffix=None):
     elif isinstance(resource, Registration) and (clean_suffix in ('', 'comments', 'links', 'components', 'resources',)) and waffle.flag_is_active(request, features.EMBER_REGISTRIES_DETAIL_PAGE):
         return use_ember_app()
 
+    elif isinstance(resource, Registration) and clean_suffix.startswith('metadata') and waffle.flag_is_active(request, features.EMBER_REGISTRIES_DETAIL_PAGE):
```

Comment:
Should we add the null check `and clean_suffix` before `clean_suffix.startswith(...` (similar to the other changes)?

Reply:
Ummm maybe we should.

Reply:
Yep, I think the one above didn't check because the next check is `clean_suffix in (...)` which doesn't break if `None`